src/ptvsd/adapter/debuggee.py

```diff
@@ -80,6 +80,47 @@ def spawn_and_connect(request):
             before_accept=lambda address: _parse_request_and_spawn(request, address),
         )
 
+def connect_to_process(request):
```

Comment:
Let's call it `attach_by_pid` or something similar - in line with whatever is being used by the tests currently. As is, it's a bit unclear that either this function, or the worker below, uses "processId" to inject.

Also, looks like `_parse_request_and_inject` could be nested?